src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old

```diff
@@ -2765,6 +2765,7 @@
 "/cloudresourcemanager:v1beta1/key": key
 "/cloudresourcemanager:v1beta1/quotaUser": quota_user
 "/cloudresourcemanager:v1beta1/cloudresourcemanager.projects.create": create_project
+"/cloudresourcemanager:v1beta1/cloudresourcemanager.projects.create/useLegacyStack": use_legacy_stack
```

Comment:
This was a backup made by the apiary_names_update.sh script for comparison purposes. I'll remove it.